tests/unit/song_tests.js

```diff
@@ -2,41 +2,41 @@ describe('song tests', function() {
     var songObj = {"image":"../core/assets/2015-best-music.png","songTitle":"love love enemies love","songAuthor":"Smiley","songLength":74,"songListened":82};
     var song;
     beforeEach(function() {
-      song = new Song(songObj);
+      this.song = new Song(songObj);
```

Comment:
Am înțeles, dar înainte era fără this și a zis Gabi să pun cu this. Citez: "You can do here(to avoid the risk of sharing "state" between tests) this.song = new Song() and in every test you can access it as this.song".